### ISSUE TITLE
Menu Button Not Working

### ISSUE THREAD

**Issue by ndtvchris** [cortex-lab/phy#1183]
Hello,

I am trying to use phy to curate data from kilosort2. The functionality is great, but I can't seem to use the menu button (stack of 3 horizontal lines) in any of the views, and I'm not sure why. I am using phy on a MacBook Pro. 

Thank you!

Sincerely,
Christopher Nguyen

**Comment by ndtvchris** [comment]
No, but I'm starting to think it's a Mac limitation? My labmate uses it on her Linux machine and has full functionality.

**Comment by rossant** [comment]
Closing in favor of #1163.
